src/app/shared/components/editor/component.ts

```diff
@@ -46,6 +46,7 @@ export class EditorComponent implements OnInit {
     scrollbar: {vertical: 'hidden'},
     hideCursorInOverviewRuler: true,
     renderLineHighlight: 'none',
+    wordWrap: 'on',
```

Comment:
I'm not sure, if this is easily fixable, but I noticed that the indentation is not given once the line is too long, see:

![editor](https://user-images.githubusercontent.com/19547196/96129782-5aeb1980-0ef7-11eb-9938-d289cc9e7461.PNG)

On the other hand if you already have like 4-5 indentations, the text will be very short.... but for a few indentations it might look a bit more organized? 🤔 

Also I noticed, that the editor has some "problems" when you directly want to place the cursor via mouse. It looks like it is on the given place, but in reality it isn't.... But this is probably not part of this PR 😅

Reply:
> I'm not sure, if this is easily fixable, but I noticed that the indentation is not given once the line is too long, see:

This is expected in my opinion.

Reply:
> Also I noticed, that the editor has some "problems" when you directly want to place the cursor via mouse. It looks like it is on the given place, but in reality it isn't.... But this is probably not part of this PR 

Yep, I will report an issue for that.

Reply:
Okay, I just noticed it and was wondering if it is expected or if it might look a bit more organized to align everything the same (and on the other hand noticed that many indentations might cause very short text in a row) 👍 

Though this is not a blocker :)

Reply:
Sure, I think this may be caused by the removal of the !important rules and refactoring that we did some time ago.